### DISCUSSION TITLE
Problem der Erkennung des VCA im Multiplayer

### DISCUSSION THREAD

**Discussion by TheEvil5284** [Mogli12/KeyboardSteer#732] [General]
Moin,

wir haben folgendes Problem.
Die aktuelle Version 9.0.2.7 liegt auf unserem Dediserver, wir laden uns von dem die aktuellen Mods herunter.

Das Spiel aber erkennt nicht dass die Mod installiert ist und folglich kommen wir nicht auf den Server. Schmeißen
wir die VCA Mod wieder runter, kommen wir rauf.

Gruß
TheEvil

**Comment by Mogli12** [comment]
Hallo,

das hat nichts mit dem Mod zu tun. Allerdings sollte VCA demnächst im ModHub erscheinen.

Viele Grüße
